deployment/ds-hostnet-split/03-envoy.yaml

```diff
@@ -57,7 +57,20 @@ spec:
         lifecycle:
           preStop:
             exec:
-              command: ["wget", "-qO-", "http://localhost:9001/healthcheck/fail"] 
+              command: ["wget", "-qO-", "http://localhost:9001/healthcheck/fail"]
+      - name: statsd-sink
+        image: prom/statsd-exporter:v0.6.0
+        command: 
+            - "/bin/statsd_exporter"
```

Comment:
Just curious: envoy 1.10.0 has just been released (like 2 hours ago) and supports histograms in prometheus export now (https://github.com/envoyproxy/envoy/pull/5601 you shared it on slack a few days ago :-D) . Does the sidecar add more metrics if envoy 1.10 is used?

Also, I'm not sure when contour plans to upgrade to envoy 1.10, so even if it does, this can be totally relevant :-)

Reply:
@rata yes Envoy v1.10 has Histograms enabled so we can remove the statsd forwarder bit. I'm not sure when we'll upgrade to v1.10, we typically wait a bit to let any issues shake out. I'd like to still merge this for v1.9.1 and then we'll update with a future PR once we move to envoy v1.10

Reply:
Thanks a lot for the info! :)